src/styles/styles.js

```diff
@@ -2874,6 +2874,37 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    joinRoomPromptContainer: {
+        maxHeight: 92,
+        height: 92,
+        borderRadius: 18,
+        padding: 20,
+        backgroundColor: themeColors.overlay,
+        display: 'flex',
+        flexDirection: 'row',
+    },
+
+    joinRoomPromptTextContainer: {
+        paddingTop: 3.5,
+        paddingBottom: 3.5,
+    },
+
+    joinRoomPromptButtonContainer: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'flex-end',
+        alignItems: 'center',
+    },
+
+    joinRoomButton: {
+        maxHeight: 40,
```

Comment:
Why do we need to add a maxHeight? I feel like our standard button style should suffice here, which sits perfectly at 40px (or it should).